src/app/app.cpp

```diff
@@ -70,8 +70,10 @@ int App::run(int argc, char** argv)
     }
 #endif
 
+    //! NOTE Just for get version
```

Comment:
From my perspective, It looks strange. At first, I thought that we are creating a QApplication here
May be we should use static methods in Application like it was before with MUVersion
What do you think?

Reply:
There is a problem with static methods: if we add them to the implementation class, we will have to use the “internal” class, and if we add them to the interface, it will look strange...

If you don't like this solution, I'll change it to another one now